rosbag2_transport/test/rosbag2_transport/test_play_loop.cpp

```diff
@@ -43,6 +43,8 @@ TEST_F(RosBag2PlayTestFixture, play_bag_file_twice) {
   const size_t read_ahead_queue_size = 1000;
   const float rate = 1.0;
   const bool loop_playback = false;
+  double clock_publish_frequency = 0.0;
+  const float delay = 1.0;
```

Comment:
Can you also test for an invalid delay value?

Reply:
I added tests for delay in https://github.com/ros2/rosbag2/pull/789/commits/dfee76ad21b4e396eef0e8d2f12bd1f8b084ea4e.